ipynb/energy/EnergyModel_ClusterEnergy.ipynb

```diff
@@ -106,12 +106,12 @@
     "    \"emeter\" : {\n",
     "        'instrument' : 'hwmon',\n",
     "        'conf' : {\n",
-    "            'sites' : [ 'a53', 'a57' ],\n",
+    "            'sites' : [ 'BOARDLITTLE', 'BOARDBIG' ],\n",
```

Comment:
So I guess the issue is that on JunoR0 boards, running a v3.18 LSK kernel, HWMON channels are named BOARDLITTLE and BOARDBIG... which means that some patches have been backported from mainline. Can you please check this? If that's the case we should actually get this patch in. ;-)